.github/workflows/core_build_publish.yml

```
@@ -69,13 +69,31 @@ jobs:
           dotnet-version: "5.0.100"
       - name: Restore
         run: dotnet restore
+      - name: Install trx2junit
+        run: dotnet tool install --global trx2junit
       - name: Build
         run: dotnet build --configuration Release --no-restore
         env:
           GIT_COMMIT: ${{ github.sha }}
       - name: Test
-        run: dotnet msbuild /t:RunTests /p:Configuration=Release
+        run: dotnet msbuild /t:RunTestsWithCoverage
         working-directory: test
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        if: always()
+
+      - name: Convert Test Results to jUnit
+        run: trx2junit --output TestResults $(find -name '*.trx')
```

```suggestion
        run: find -name '*.trx' -exec trx2junit --output TestResults {} +
```
Otherwise path splitting can happen.

---

The TRX files don't have space in the name nor the path where everything lands. We're safe here.

---

That's a bold assumption which I'd rather we didn't rely on.